src/libertem/api.py

```diff
@@ -95,7 +96,7 @@ def load(self, filetype: str, *args, **kwargs) -> DataSet:
     load.__doc__ = load.__doc__ % {"types": ", ".join(filetypes.keys())}
 
     def create_mask_job(self, factories, dataset, use_sparse=None,
-                        mask_count=None, mask_dtype=None, dtype=None):
+                        mask_count=None, mask_dtype=None, dtype=None) -> np.ndarray:
```

Comment:
`np.adarray` is not the correct type here, please compare with the docstring. You can also have a look at the code of the method itself: find the `return` statements and have a look at what kind of object is returned